core/src/main/scala/dagr/core/tasksystem/Pipe.scala

```diff
@@ -72,6 +73,19 @@ trait Pipe[In,Out] extends ProcessTask {
     Pipes.chain(this, new RedirectToFile[Out](path, append=true))
   }
 
+  /** Writes the standard error stream to the given path. */
+  def >!(path:Path): Pipe[In,Out] = {
+    Pipes.chain(this, new RedirectErrorToFile[Out, Out](path, append=false))
+  }
+
+  /** Appends the standard error stream to the given path. */
+  def >>!(path:Path): Pipe[In,Out] = {
+    Pipes.chain(this, new RedirectErrorToFile[Out, Out](path, append = true))
+  }
+
+  /** Discards the standard error stream. */
+  def discardError() = { >!(Io.DevNull) }
```

Comment:
Do you want this to return `this` too so you can `new Foo().discardError() | new Bar()`?